devito/dle/blocking_utils.py

```diff
@@ -134,43 +139,46 @@ def optimize_unfolded_tree(unfolded, root):
     """
     processed = []
     for i, tree in enumerate(unfolded):
+        assert len(tree) == len(root)
         otree = []
-        stmts = []
+        oroot = []
```

Comment:
I wanted to, but I start breaking too many lines into two if I expand these names, so I preferred the contracted form. I'll see if I can rethink them though, because you're right, ``stmts`` is quite obvious, but ``oroot`` is definitely not